src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,93 @@
-export const Pagination = () => {};
+import { getNumbers } from '../../utils';
+
+interface IPagination {
+  perPage: number;
+  total: number;
+  page: number;
+  setPage: React.Dispatch<React.SetStateAction<number>>;
+  getCurrentItems: () => string[];
+}
+
+export const Pagination = ({ page, setPage, perPage, total, getCurrentItems }: IPagination) => {
+
+  // Número total de páginas
+  const listNumbers = Math.ceil(total / perPage);
+
+  // Geração dos números de páginas
+  const pageNumbers = getNumbers(1, listNumbers);
+
+  // Função para obter os itens da página atual
+
+
+  // Mudar para a próxima página
+  const goToNextPage = () => setPage((prev) => Math.min(prev + 1, listNumbers));
+
+  // Mudar para a página anterior
+  const goToPreviousPage = () => setPage((prev) => Math.max(prev - 1, 1));
+
+  // Mudar para uma página específica
+  const goToPage = (pageNumber: number) => setPage(pageNumber);
```

Comment:
The 'goToPage' function is redundant as it's simply wrapping the 'setPage' function without adding any additional logic. You can directly use 'setPage' where 'goToPage' is being used.